keps/sig-multicluster/2149-clusterid/README.md

```diff
@@ -324,15 +396,21 @@ Contains a unique identifier for the containing cluster.
 
 ##### Uniqueness
 
-*   The identifier **must** be unique within the ClusterSet to which its cluster belongs for the duration of the cluster’s membership.
-*   The identifier **should** be unique beyond the ClusterSet within the scope of expected use.
-*   The identifier **may** be globally unique beyond the scope of its ClusterSet.
-*   The identifier **may** be unique beyond the span of its cluster’s membership and lifetime.
+*   The identifier **must** be unique within the ClusterSet to which its cluster
+    belongs for the duration of the cluster’s membership.
+*   The identifier **should** be unique beyond the ClusterSet within the scope
+    of expected use.
+*   The identifier **may** be globally unique beyond the scope of its
+    ClusterSet.
+*   The identifier **may** be unique beyond the span of its cluster’s membership
+    and lifetime.
 
 
 ##### Lifespan
 
-*   The identifier **must** exist and be immutable for the duration of a cluster’s membership in a ClusterSet, and as long as a `clusterset.k8s.io` property referring to that cluster in that ClusterSet exists.
+*   The identifier **must** exist and be immutable for the duration of a
+    cluster’s membership in a ClusterSet, and as long as a `clusterset.k8s.io`
+    property referring to that cluster in that ClusterSet exists.
 *   The identifier **must** exist for the lifespan of a cluster.
```

Comment:
I do not understand this.  Just above you said "for the duration of..." and now you are saying "forever".  I think this one may be vestigial?

Reply:
Agreed, needs cleanup